distsql/request_builder.go

```diff
@@ -283,6 +294,17 @@ func (builder *RequestBuilder) SetResourceGroupTag(sc *stmtctx.StatementContext)
 	return builder
 }
 
+func (builder *RequestBuilder) setMatchStoreLabels() {
+	if builder.IsStaleness && builder.txnScope != kv.GlobalTxnScope {
+		builder.MatchStoreLabels = []*metapb.StoreLabel{
+			{
+				Key:   placement.DCLabelKey,
+				Value: builder.txnScope,
```

Comment:
I remember we proposed to make txnScope to `Local/Global` rather than the label values to reduce the burden of users. may we rename txnScope to others in here? actually, it is a buildin label value use to dispatch requests.

Reply:
updated.